BUILD.bazel

```diff
@@ -1 +1,3 @@
 # This file exists so the root of our repository, e.g. the root `Cargo.toml`, can be referenced by Bazel.
+
+exports_files(["Cargo.toml"])
```

Comment:
What does this do? Does it write out a new Cargo.toml file?

Reply:
It just makes the `Cargo.toml` file at the root of our repo visible to other build rules! We need this because individual crates inherit lints from the workspace